configs/templates/skylab-dev/spack.yaml

```diff
@@ -43,7 +43,7 @@ spack:
     - fckit@0.9.5
     # DH* fake version number
     - fiat@1.0.0
-    - flex@2.6.4
+    #- flex@2.6.4
```

Comment:
Note to self. Add a comment here that specifying the bison version leads to duplicate packages being built (some using the default/OS bison, some the one requested here). Do after CI tests ran.